### PR TITLE
snap: fix errors in AddImplicitSlots

### PR DESCRIPTION
**Pull request by zyga**

### CODE REVIEW

File: snap/implicit.go [hunk 1]
```diff
@@ -29,13 +29,13 @@ var implicitSlots = []string{
 	"network-bind",
 	"network-control",
 	"network-observe",
-	"snap-control",
+	"snapd-control",
 	"system-observe",
 	"timeserver-control",
 	"timezone-control",
 	// XXX: those two should perhaps not be added by default
```
Comment (niemeyer) [inline snap/implicit.go]: Please drop the comment. We don't want people to have to manually say their desktop snaps should actually work.

File: snap/implicit.go [hunk 2]
```diff
@@ -29,13 +29,13 @@ var implicitSlots = []string{
...
 	"unity7",
-	"x",
+	"x11",
```
Comment (niemeyer) [inline snap/implicit.go]: Can we run through that list in tests and double-check the names actually exist?

Comment (zyga) [inline snap/implicit.go]: Yes, good idea.

File: snap/implicit.go (hunk 1 above)
Comment (zyga) [inline snap/implicit.go]: Done

File: snap/export_test.go [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package snap
+
+var ImplicitSlots = implicitSlots
```
Comment (niemeyer) [inline snap/export_test.go]: s/ImplicitSlots/ImplicitSlotsForTests/

**Comment by niemeyer** [comment]
One comment, and LGTM.

**Pull request merged**